jackbox_patcher/assets/l10n/app_de.arb

```diff
@@ -98,7 +98,7 @@
     "finalizing": "Fast fertig...",
     "unknown_error": "Unbekannter Fehler",
     "contact_error": "Melde dich bitte bei Erizzle#0001 auf Discord",
-    "game_patch_unavailable": "Patch nicht verfügbar",
+    "game_patch_unavailable": "Patch ist nicht verfügbar",
```

Comment:
Let's keep "Patch nicht verfügbar"

Reply:
Either the patch **is** available or not. So using *ist* should be better and correct here.